### PR TITLE
Establish KiCAD files and include important project information

### PR DESCRIPTION
**Pull request by ericmutton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,38 @@
 # Cobalt-Cowboy
-An ESP32-C3 based radio running OpenRTX Firmware
+
+A RISC-V-based radio running OpenRTX Firmware
+
+The goal of this project is to make available a hackable radio development platform built using open-source hardware and firmware, reducing the copyright and patent overhead required to build radio equipment and technology.
+
+The intended users of this project include licenced Amateur Radio operators and students.
+
+This repo includes the PCB and Schematic and associated [KiCAD](https://www.kicad.org/) project files. In essence, a ESP32C3+CC1200 evaluation board built for use with the [M17 Project](https://m17project.org/) and capabilities of the [OpenRTX firmware](https://openrtx.org).
```
Comment (MStolen) [inline README.md]: I'd say something more like "This repo includes the PCB, schematic, and associated KiCAD project files *as well as a parts list and any information needed to build your own Cobalt Cowboy*..."

File: README.md [hunk 2]
```diff
@@ -1,2 +1,38 @@
...
+## License
+
+This project is released under the GNU GPL v3.
+
+See [LICENSE](LICENSE)
+
+## Credits
+
+Our thanks go to the following organizations for their efforts and advice:
+* [M17 Project](https://m17project.org/) for their digital radio protocol for data and voice
```
Comment (MStolen) [inline README.md:39]: May want to mention M17 somewhere above since that's the big digital mode we're interested in targeting.

**Review by MStolen — APPROVED** [review]
I left a few suggestions, but think this is generally pretty good.

**Comment by AbleArcher88** [comment]
Approved. Let's get this party started.

**Review by AbleArcher88 — COMMENTED** [review]
Approved

**Review by AbleArcher88 — APPROVED** [review]
Approved. Good work so far

**Pull request merged**